src/main/java/org/springframework/data/elasticsearch/repository/config/EnableElasticsearchRepositories.java

```diff
@@ -111,4 +112,10 @@
 	 * @return
 	 */
 	String elasticsearchTemplateRef() default "elasticsearchTemplate";
+
+	/**
+	 * Configures whether nested repository-interfaces (e.g. defined as inner classes) should be discovered by the
+	 * repositories infrastructure.
+	 */
+	boolean considerNestedRepositories() default false;
```

Comment:
Ok, I did it in this PR https://github.com/spring-projects/spring-data-elasticsearch/pull/91